packages/manager/modules/manager-pci-common/src/api/hook/useFeaturedDeploymentModes.ts

```diff
@@ -0,0 +1,41 @@
+import { useMemo } from 'react';
+import { TRegion } from '@/api/data';
+import { Deployment } from '@/components/deployment-tiles-input';
```

Comment:
We should import a type from an UI component in the business logic folder

Reply:
The typing is only related to UI, this call is only a helper on it since it is not a real domain object, or I could put it inside a DTO folder ?

Reply:
Yes please, to follow the separation of concerns principle